pkg/operator/sync_openshiftcontrollermanager_v311_00.go

```diff
@@ -262,16 +265,23 @@ func manageOpenShiftGlobalCAConfigMap_v311_00_to_latest(kubeClient kubernetes.In
 	if !*modified {
 		return existing, false, nil
 	}
-	updated, err := client.ConfigMaps(util.TargetNamespace).Update(existingCopy)
+
+	// we can embed input hashes on our main configmap to drive rollouts when they change; so we are including a hash of
+	// the global proxy CA map since the image signature controller access registries outside the cluster
+	inputHashes, err := resourcehash.MultipleObjectHashStringMapForObjectReferences(
+		kubeClient,
+		resourcehash.NewObjectRef().ForConfigMap().InNamespace(util.TargetNamespace).Named("openshift-global-ca"),
+	)
```

Comment:
i don't think you need to do this (nor should you, since you're adding a random key to the CA configmap which can then end up getting mounted into the pod).

just add the configmap to the list that already exists here:
https://github.com/openshift/cluster-openshift-controller-manager-operator/blob/master/pkg/operator/sync_openshiftcontrollermanager_v311_00.go#L195-L199

that code is computing a hash based on all the configmaps we care about, and storing that hash in whatever "main configmap" is and applying the change to the main configmap.  If that results is an update to mainconfigmap, force rollout is returned (based on my somewhat superficial skimming of the code).

Reply:
suggestion pushed @bparees thanks